### ISSUE TITLE
Question: UsingHead always returns 0 for Content-Length header even when explicitly specified

### ISSUE THREAD

**Issue by pmcilreavy** [WireMock-Net/WireMock.Net#126]
The response to a HEAD request should be the length of the file (without the actual body) so something like this (below) should return 1024 in the Content-Length response header but it always returns 0. Something internally must be overwriting my value of "1024" back to "0".

``` c#
server
    .Given(Request.Create().WithPath($"/SomeFile.zip").UsingHead())
    .RespondWith(Response.Create()
        .WithHeader("Content-Length", "1024")
    );
```

**Comment by StefH** [comment]
@pmcilreavy Does this answer your question?

**Comment by StefH** [comment]
Closing

**Comment by o7g8** [comment]
@StefH the documentation https://msdn.microsoft.com/en-us/library/78h415ay(v=vs.110).aspx says 
```
Tests whether the specified HTTP header can be set for the request.
```
But in this example the code attempts to set in the _response_ 
Why shouldn't it be possible for a _mock_ to specify an arbitrary value of `Content-Length`?

**Comment by StefH** [comment]
Solved, see NuGet 1.0.3.18
